webpages/assets/main.css

```diff
@@ -47,7 +50,7 @@ h1 {
 
 .main {
   display: flex;
-  height: calc(100% - 60px);
+  height: calc(100vh - 60px);
```

Comment:
This causes the license page background to change colour when scrolling, Do you remember why it was changed?

Reply:
hmm, i can't reproduce.